src/utilities/test-utils.test.ts

```diff
@@ -2,6 +2,27 @@ import { TestUtils } from "./test-utils";
 import faker from "faker";
 
 describe("TestUtils", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region randomCase
+    // -----------------------------------------------------------------------------------------
+
+    describe("randomCase", () => {
+        test("it randomizes case of string", () => {
+            // Arrange & Act
+            let fixture = "";
+            for (let i = 0; i < 20; i++) {
+                fixture += "a";
+            }
+            const result = TestUtils.randomCase(fixture);
```

Comment:
Since there are distinct steps here (`Arrange` would be the setup of the input string, and `Act` is the actual call to the system under test), could we separate those out with separate comments?

Reply:
I think with those shortened `repeat` method, I can just inline setup